unfoldingWord/translationCore
# Localized Components

#### This pull request addresses:

This update provides localization for most of the components within the application.
There are still a number of items that need localization. However, these will require some major refactoring therefore I'll put those in a separate PR.

This makes progress on #2265.

#### How to test this pull request:

You won't be able to see any changes by running the app since all of the localization are defaulting to English at the moment. Therefore it's sufficient to simply ensure the application opens properly without crashing. You could also check in the console to ensure there are no errors about missing translation keys.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3699)
